src/infrastructure/note.repository.ts

```diff
@@ -30,16 +58,25 @@ export default class NoteRepository implements NoteRepositoryInterface {
     const noteData = await this.noteStorage.getNoteById(id);
 
     /**
-     * If note data not exists
+     * If note data in storage exists
      */
-    if (!noteData) {
-      return null;
+    if (noteData) {
+      return {
+        id: noteData.id,
+        title: noteData.title,
+        content: noteData.content,
+      };
     }
 
+    /**
+     * Get note data from API
+     */
+    const noteResponse = await this.transport.get<GetNoteResponse>('/note/' + id);
+
```

Comment:
and you need to set it to the storage now

Reply:
done